src/script/lua_api/l_object.cpp

```diff
@@ -907,13 +907,65 @@ int ObjectRef::l_get_yaw(lua_State *L)
 	NO_MAP_LOCK_REQUIRED;
 	ObjectRef *ref = checkobject(L, 1);
 	LuaEntitySAO *co = getluaobject(ref);
-	if (co == NULL) return 0;
-	// Do it
+	if (!co) return 0;
+
 	float yaw = co->getYaw() * core::DEGTORAD;
 	lua_pushnumber(L, yaw);
 	return 1;
 }
 
+// set_pitch(self, radians) (X-AXIS)
+int ObjectRef::l_set_pitch(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	LuaEntitySAO *co = getluaobject(ref);
+	if (!co) return 0;
+	float pitch = luaL_checknumber(L, 2) * core::RADTODEG;
+
```

Comment:
Best have the empty line immediately after the `return;`
Below too.

Reply:
Would this suffice? Any improvements? Looks like too many empty lines:
```
int ObjectRef::l_set_yaw(lua_State *L)
{
	NO_MAP_LOCK_REQUIRED;
	ObjectRef *ref = checkobject(L, 1);
	LuaEntitySAO *co = getluaobject(ref);
	if (!co) 
	    return 0;
	
	float yaw = luaL_checknumber(L, 2) * core::RADTODEG;

	co->setYaw(yaw);
	return 0;
}
```

Reply:
Agreed, just one empty line is enough, often we add an empty line after a 'return'.